phpBB/styles/prosilver/template/mcp_forum.html

```diff
@@ -1,7 +1,7 @@
 <!-- INCLUDE mcp_header.html -->
 
 <!-- DEFINE $CUSTOM_FIELDSET_CLASS = 'forum-selection2' -->
-<!-- INCLUDE jumpbox.html -->
+<!-- INCLUDE mcp_jumpbox.html -->
```

Comment:
eitherway I would still do the action bar div external to the file
`
<div class="action-bar actions-jump">
<!-- INCLUDE mcp_jumpbox.html -->
</div>
`